transformer/mappings.py

```diff
@@ -72,34 +72,6 @@ class ArchivesSpaceExtentToExtent(odin.Mapping):
     )
 
 
-class ArchivesSpaceSubnoteToSubnote(odin.Mapping):
-    from_obj = ArchivesSpaceSubnote
-    to_obj = Subnote
-
-    @odin.map_field(from_field='jsonmodel_type')
-    def type(self, value):
-        if value:
-            if value in ['note_orderedlist', 'note_definedlist']:
-                return Subnote(type=value.split('note_')[1], content=self.source.items)
-            elif value == 'note_bibliography':
-                data = []
-                data.append(Subnote(type='text', content=self.source.content))
-                data.append(Subnote(type='orderedlist', content=self.source.items))
-                return data
-            elif value == 'note_index':
-                data = []
-                l = [{'label': i.get('type'), 'value': i.get('value')} for i in self.source.items]
-                data.append(Subnote(type='text', content=self.source.content))
-                data.append(Subnote(type='definedlist', content=l))
-                return data
-            elif value == 'note_chronology':
-                m = [{'label': i.get('event_date'), 'value': ', '.join(i.get('events'))} for i in self.source.items]
-                return Subnote(type='definedlist', content=m)
-            else:
-                return Subnote(type='text', content=self.source.content
-                               if isinstance(self.source.content, list) else [self.source.content])
-
-
```

Comment:
I removed this because this functionality has now been moved to within the `ArchivesSpaceNoteToNote `mapping